hedera-node/src/main/java/com/hedera/services/fees/calculation/consensus/txns/UpdateTopicResourceUsage.java

```diff
@@ -46,33 +46,36 @@ public boolean applicableTo(TransactionBody txn) {
     }
 
     @Override
-    public FeeData usageGiven(TransactionBody txn, SigValueObj sigUsage, StateView view) throws InvalidTxBodyException {
-        try {
-            MerkleTopic merkleTopic = view.topics().get(MerkleEntityId.fromTopicId(txn.getConsensusUpdateTopic().getTopicID()));
-            long rbsIncrease = getUpdateTopicRbsIncrease(
-                    txn.getTransactionID().getTransactionValidStart(),
-                    JKey.mapJKey(merkleTopic.getAdminKey()),
-                    JKey.mapJKey(merkleTopic.getSubmitKey()),
-                    merkleTopic.getMemo(),
-                    merkleTopic.hasAutoRenewAccountId(),
-                    lookupExpiry(merkleTopic),
-                    txn.getConsensusUpdateTopic());
-            return getConsensusUpdateTopicFee(txn, rbsIncrease, sigUsage);
-        } catch (Exception illegal) {
-            log.warn("Usage estimation unexpectedly failed for {}!", txn, illegal);
-            throw new InvalidTxBodyException(illegal);
+    public FeeData usageGiven(TransactionBody txnBody, SigValueObj sigUsage, StateView view) throws InvalidTxBodyException {
+        if(txnBody == null) {
+            throw new InvalidTxBodyException("consensusCreateTopic field not available for Fee Calculation");
+        } else {
+            try {
```

Comment:
It's also good to add a check (maybe not needed, but doesn't hurt to be safe than sorry) of null for the state view.